djstripe/forms.py

```diff
@@ -168,7 +168,8 @@ def _post_clean(self):
         try:
             self._stripe_data = models.WebhookEndpoint._api_create(
                 url=url,
-                api_version=self.cleaned_data["api_version"] or None,
+                api_version=self.cleaned_data["api_version"]
+                or settings.djstripe_settings.STRIPE_API_VERSION,
```

Comment:
This feels wrong to me. The configured version and the account's default version may be mismatched. -1 on this change.

Reply:
I may have miscommunicated but  `settings.djstripe_settings.STRIPE_API_VERSION` has to match the `Stripe Dashboard Configured Version`. Without that `dj-stripe` will not work so the scenario you're referring to cannot happen in my opinion. `Stripe` does allow `webhooks` to have their own `version` and this change is simply making that choice explicit.